src/styles/main.scss

```diff
@@ -0,0 +1,149 @@
+$color1: #25283d;
+
+@mixin wh($width, $height) {
+  width: $width;
+  height: $height;
+}
+
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
you shouldn't reset indentation on * selector, reset it directly for element you need